medusa/subtitle_providers/legendastv.py

```diff
@@ -477,3 +477,12 @@ def download_subtitle(self, subtitle):
 
         # extract subtitle's content
         subtitle.content = fix_line_ending(subtitle.archive.content.read(subtitle.name))
+
+
+def raise_for_status(r):
+    # When site is under maintaince and http status code 200.
+    if 'Em breve estaremos de volta' in r.text or r.status_code == 503:
+        # For now raise Timeout until the upstream PR gets merged that catches ServiceUnavailable
+        raise Timeout
```

Comment:
I couldn't find a way to avoid to catch it. 
Another solution is to raise `ProviderError` but that will show a traceback with `Unexpected error in provider`